docs/3-retrieving-data-as-lists.md

```diff
@@ -54,21 +60,23 @@ const queryList = af.database.list('/items', {
 To get the list in realtime, create a list binding as a property of your component or service.
 Then in your template, you can use the `async` pipe to unwrap the binding.
 
+Update `/src/app/app.component.ts` to import `FirebaseListObservable` from angularfire2 and iterate thru the list once data is retrieved. Also note the change in attribute `templateUrl` to inline `template` below.
+
 ```ts
 import {Component} from '@angular/core';
 import {AngularFire, FirebaseListObservable} from 'angularfire2';
 
 @Component({
-  selector: 'app',
-  templateUrl: `
+  selector: 'app-root',
+  template: `
   <ul>
     <li *ngFor="let item of items | async">
-      {{ item.name }}
+       {{ item.$value }}
```

Comment:
I don't like `item.$value` here. This field is only useful for primitives and that's not the common use case. `item.name` represents an arbitrary value in the field. Let's put in something more useful for debugging and initial output? How about:  `{{ item | json }}`